client/my-sites/email/controller.js

```diff
@@ -75,6 +75,7 @@ export default {
 				<EmailProvidersComparison
 					comparisonContext="email-purchase"
 					selectedDomainName={ pageContext.params.domain }
+					context={ pageContext.query.context }
```

Comment:
I think we have a naming problem here. We're adding a new `context` prop but we already have this `comparisonContext` prop. It's not clear to me what's the difference between the two.

Reply:
I wondered the same @stephanethomas, my intention was not to change the original comparisonContext, as it's being used to track different events. So I tried to avoid regression by adding a new prop that would not interfere with other tracking methods. I am unsure if we should merge those two parameters... With something like

`pageContext.query.context ?? "email-purchase"`

Reply:
Finally I've merged both props.